components/output/mcp4461.rst

```diff
@@ -0,0 +1,188 @@
+MCP4461 Component
+=================
+
+.. seo::
+    :description: Instructions for setting up MCP4461 outputs on the ESP.
+    :image: mcp4461.jpg
+
+.. _mcp4461-component:
+
+The MCP4461 output component enables the use of an 8‑bit external quad digital potentiometer/rheostat via I²C. See the `MCP4461 Datasheet <https://www.microchip.com/en-us/product/mcp4461>`__ for more information.
+Device default address is ``0x2C``.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+
+    # Set a global I²C connection
+    i2c:
+      sda: 21
+      scl: 22
+      scan: true
+
+    # Set the component with default (address: 0x2C / global I²C)
+    mcp4461:
+      - id: mcp4461_output
+
+Configuration variables:
+------------------------
+
+- **id** (*Optional*, :ref:`config-id`): The id to use for this output component.
+- **address** (*Optional*, int): Manually specify the I2C address of
+  the digipot. Defaults to ``0x2C``.
+- **disable_wiper_0** (*Optional*, bool): Disable wiper 0. Defaults to ``false``.
+- **disable_wiper_1** (*Optional*, bool): Disable wiper 1. Defaults to ``false``.
+- **disable_wiper_2** (*Optional*, bool): Disable wiper 2. Defaults to ``false``.
+- **disable_wiper_3** (*Optional*, bool): Disable wiper 3. Defaults to ``false``.
+  
+MCP4461 Output
+**************
+
+The MCP4461 output component exposes 8 MCP4461 (wiper-)channels of a global :ref:`MCP4461 <mcp4461-component>` as float outputs.
+
+.. code-block:: yaml
+
+    # Individual outputs
+    # A-D are volatile wipers 0-3
+    # E-H are nonvolatile wipers 0-3
+    # (AE, BF, CG, DH belonging together)
+    output:
+    - platform: mcp4461
+      id: digipot_channel_0
+      mcp4461_id: mcp4461_output
+      channel: A
+      initial_value: 0.128 // always initialize volatile wiper 0 with wiper @ medium resistance range on start
```

Comment:
Careful -- this is not a comment in yaml and will result in unexpected behavior.
```suggestion
      initial_value: 0.128  # always initialize volatile wiper 0 with wiper @ medium resistance range on start
```